.gitignore

```diff
@@ -2,3 +2,4 @@ vendor
 *.swp
 book
 cctool
+!cctool/
```

Comment:
Why is this needed?

Reply:
i assume the initial intent was to not include cctool binary. But by doing this that regex also matches the dir name and ignores any modifications to files children of /cmd/cctool

Reply:
ah yeah whoops